set.mm

```diff
@@ -265783,18 +265769,16 @@ as element of (the base set of) ` ( ( 1 ... n ) Mat R ) `.
     $( The matrix ring has the same base set as its underlying group.
        (Contributed by Stefan O'Rear, 4-Sep-2015.) $)
     matbas $p |- ( ( N e. Fin /\ R e. V ) -> ( Base ` G ) = ( Base ` A ) ) $=
-      ( cfn wcel wa cbs cfv cnx cmulr cotp cmmul co cop c1 c3 csts eqid basendx
-      matval fveq2d baseid 1re ltneii mulrndx neeqtrri eqnetri setsnid syl6reqr
-      1lt3 ) DHIBEIJZAKLCMNLZBDDDOPQZRUAQZKLCKLUOAURKABUQCDEFGUQUBUDUEUQUPKCUFM
-      KLSUPUCSTUPSTUGUNUHUIUJUKULUM $.
+      ( cfn wcel wa cbs cfv cnx cmulr cotp cmmul co cop csts eqid matval fveq2d
+      baseid basendxnmulrndx setsnid syl6reqr ) DHIBEIJZAKLCMNLZBDDDOPQZRSQZKLC
```

Comment:
Yes, basendx is replaced by basendxnmulrndx, which is not discouraged (and also not based on discouraged theorems). Therefore, the replacement/shortening is OK.

Reply:
Looks fine to me!

One naive related question would be, since `basendx` is discouraged, shouldn't also `df-base` be discouraged?
Indeed what we want to discourage here is the direct use of the hidden slot definition.

Reply:
> One naive related question would be, since `basendx` is discouraged, shouldn't also `df-base` be discouraged? Indeed what we want to discourage here is the direct use of the hidden slot definition.

Yes, this is also my opinion. But there are some more theorems which directly depend on the specific slot definition, and it is maybe difficult to draw the line between discouraged and not discourade theorems (should ~basendxnmulrndx also be discouraged?). But this is off topic here...

Reply:
@tirix - Can you (or someone else) create a *new* issue proposing that df-base be discouraged, so we can track that as its own issue?